template/base/src/pages/_app.tsx

```diff
@@ -1,5 +1,5 @@
 import "../styles/globals.css";
-import type { AppProps } from "next/app";
+import type { AppProps } from "next/dist/shared/lib/utils";
 
 function MyApp({ Component, pageProps }: AppProps) {
```

Comment:
I was actually thinking of making this an arrow function like we have in most other places. 
```ts
import type { AppType } from "next/dist/shared/lib/utils";

const MyApp: AppType = ({ Component, pageProps }) => { ... }
export default MyApp;
```

What do you think?

Reply:
Yeah, I think it will be cool, so that the code will be consistent throughout the project.

Reply:
> If you want then I can just add those changes to this PR as well.

Yes, please do!